.github/workflows/test.yml

```diff
@@ -0,0 +1,48 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 5
+      matrix:
+        python-version: ['2.7', '3.5', '3.6', '3.7', '3.8', 'pypy3']
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Get pip cache dir
+      id: pip-cache
+      run: |
+        echo "::set-output name=dir::$(pip cache dir)"
+
+    - name: Cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.pip-cache.outputs.dir }}
+        key:
+          ${{ matrix.python-version }}-v1-${{ hashFiles('**/setup.py') }}
+        restore-keys: |
+          ${{ matrix.python-version }}-v1-
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install --upgrade tox tox-gh-actions
+
+    - name: Tox tests
+      run: |
+        tox -v
+
+    - name: Upload coverage
+      uses: codecov/codecov-action@v1
+      with:
+        name: Python ${{ matrix.python-version }}
```

Comment:
The original command for Travis has a bunch of options which aren't used now:
```
codecov --required -X gcov fix pycov -f coverage.xml --flags ${TOXENV//-/ }
```

I've not looked those up, but it doesn't seem to have affected coverage? In fact, it's gone up a bit.

I don't think there's analogues for all those options with the action, but at least `flags` can be added in the `with:` section:

https://github.com/codecov/codecov-action

Reply:
Right, I looked at them, I only see `--required` easy to reply, with `fail_ci_if_error = true`.

`-X` to disable a feature seems to be not supported by the action.
`coverage.xml` is automatically found already.
`--flags` with the tox environment isn't useful anymore since in the new GHA test workflow the test matrix doesn't use TOXENV anymore, so couldn't be used. Unless I'm misunderstanding the notation of `${TOXENV//-/ }`? Does this make sense to you?

Reply:
I'm also not sure what `${TOXENV//-/ }` means, but I don't see any `$TOXENV` actually used in the old Travis CI file, so probably fine to skip 👍